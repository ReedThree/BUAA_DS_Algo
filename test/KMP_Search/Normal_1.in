0112234567890987654311224561123122456353415134151351531311112201122345678909876543112245611231224563534151341513515313111122
1122