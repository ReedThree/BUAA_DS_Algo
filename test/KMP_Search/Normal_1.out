1
20
58
63
82
120
